cpp/tests/io/json_tree.cpp

```diff
@@ -576,14 +576,14 @@ TEST_F(JsonTest, TreeRepresentation)
   // Golden sample of node categories
   std::vector<cuio_json::node_t> golden_node_categories = {
     cuio_json::NC_LIST, cuio_json::NC_STRUCT, cuio_json::NC_FN,     cuio_json::NC_STR,
-    cuio_json::NC_FN,   cuio_json::NC_LIST,   cuio_json::NC_VAL,    cuio_json::NC_VAL,
-    cuio_json::NC_VAL,  cuio_json::NC_FN,     cuio_json::NC_STR,    cuio_json::NC_FN,
-    cuio_json::NC_STR,  cuio_json::NC_FN,     cuio_json::NC_VAL,    cuio_json::NC_STRUCT,
+    cuio_json::NC_FN,   cuio_json::NC_LIST,   cuio_json::NC_STR,    cuio_json::NC_STR,
+    cuio_json::NC_STR,  cuio_json::NC_FN,     cuio_json::NC_STR,    cuio_json::NC_FN,
+    cuio_json::NC_STR,  cuio_json::NC_FN,     cuio_json::NC_STR,    cuio_json::NC_STRUCT,
     cuio_json::NC_FN,   cuio_json::NC_STR,    cuio_json::NC_FN,     cuio_json::NC_LIST,
-    cuio_json::NC_VAL,  cuio_json::NC_STRUCT, cuio_json::NC_VAL,    cuio_json::NC_STRUCT,
+    cuio_json::NC_STR,  cuio_json::NC_STRUCT, cuio_json::NC_STR,    cuio_json::NC_STRUCT,
     cuio_json::NC_FN,   cuio_json::NC_LIST,   cuio_json::NC_STRUCT, cuio_json::NC_STRUCT,
     cuio_json::NC_FN,   cuio_json::NC_STR,    cuio_json::NC_FN,     cuio_json::NC_STR,
-    cuio_json::NC_FN,   cuio_json::NC_VAL};
+    cuio_json::NC_FN,   cuio_json::NC_STR};
```

Comment:
It's removed for a specific null behaviour (null literals are identified as values now). But it might be needed in future (null literals are not checked, it might be change in future). NC_VAL does not add value now, but will be required when we add other features of json parser.